### PR TITLE
Reporting fix before consolidation of repos

### PR DESCRIPTION
**Pull request by kotharironak**
Re-porting PR (https://github.com/hypertrace/hypertrace-view-generator/pull/31/) after the consolidation of repos #15

### CODE REVIEW

File: hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java [hunk 1]
```diff
@@ -290,7 +290,7 @@ String getRequestUrl(Event event, Protocol protocol) {
     switch (protocol) {
       case PROTOCOL_HTTP:
       case PROTOCOL_HTTPS:
-        return EnrichedSpanUtils.getFullHttpUrl(event).orElse(null);
+        return EnrichedSpanUtils.getFullHttpUrl(event).orElse(event.getHttp().getRequest().getPath());
```
Comment (jcchavezs) [inline hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java:293]: Could we elaborate on why this value and not null? @buchi-busireddy @kotharironak

Comment (buchi-busireddy) [reply hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java:293]: While showing URL in the table view of spans or API traces, it makes sense to show the URL path if full URL is missing. Currently we see it empty for some spans.

**Review by buchi-busireddy — APPROVED** [review]

File: hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java (hunk 1 above)
Comment (jcchavezs) [reply hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java:293]: No, no disagreement. I was more asking on such argument as a comment in the code so in the future someone does not decide `null` is a better value.

Comment (adriancole) [reply hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java:293]: The part after this, makes it all more curious. I would advise to have coherent and fixed concepts except in span name, vs mixes ex full url or just the path part, or whatever the below returns. one can argue "requestUrl" is talking about url and a path it technically a url, but at the same time it seems this is a muddy result which can break aggregation and analysis.

```
      case PROTOCOL_GRPC:
        return event.getEventName();
```

**Pull request merged**